tests/test_obs.py

```diff
@@ -30,9 +30,8 @@
 import lsst.utils
 import lsst.utils.tests
 
-executable_dir = os.path.join(
-    lsst.utils.getPackageDir("VALIDATE_DRP"),
-    "examples")
+executable_dir = 'scripts'
+
```

Comment:
I'm a little surprised this works, I thought the tests were executed from inside `tests`.

I'm also a little worried that only two of the scripts are tested.  This would make me less worried if we had one high level script that could be configured to run any of the datasets.

Reply:
scons runs the tests from `lsst_ci/`.  

I hear you about testing only two of the scripts.  This was the current behavior.  Will keep in mind for the future as we standardize across the datasets.